its/ruling/src/test/resources/expected/python-S1481.json

```diff
@@ -1,37 +1,270 @@
 {
+'project:buildbot-0.8.6p1/buildbot/changes/hgbuildbot.py':[
+116,
+116,
+116,
+],
+'project:buildbot-0.8.6p1/buildbot/changes/mail.py':[
+89,
+281,
+],
 'project:buildbot-0.8.6p1/buildbot/clients/gtkPanes.py':[
+365,
 374,
 377,
 ],
+'project:buildbot-0.8.6p1/buildbot/clients/sendchange.py':[
+23,
```

Comment:
FP here https://github.com/SonarSource/python-test-sources/blob/master/buildbot-0.8.6p1/buildbot/clients/sendchange.py#L23
`self.username` and `self.password` are not local variables and are used here:
https://github.com/SonarSource/python-test-sources/blob/master/buildbot-0.8.6p1/buildbot/clients/sendchange.py#L45